src/app.js

```diff
@@ -19,15 +21,27 @@ const { getRemoteTabs, readError } = require(path.resolve(
 const pkg = require(path.resolve(__dirname, '../package.json'));
 const notifier = updateNotifier({ pkg });
 
+// Create a Configstore object. If conf file is empty, fill with default values instead and save to config path
+// Config path: ~/.config/configstore/check-it-out.json
+var conf = new Configstore(pkg.name, {});
+if (conf.size == 0) {
```

Comment:
If the user sets the log arguments in their custom config, but not set the themeColor, this condition will never be met and we would not set the default values.

Can we pass the default values into the instantiation of the configstore on line 26? If we define the default values there, will it overwrite the values the user defines in their json file?